packages/rum-core/src/domain/action/trackClickActions.ts

```diff
@@ -61,6 +63,7 @@ type ClickActionIdHistory = ValueHistory<ClickAction['id']>
 // Maximum duration for click actions
 export const CLICK_ACTION_MAX_DURATION = 10 * ONE_SECOND
 export const ACTION_CONTEXT_TIME_OUT_DELAY = 5 * ONE_MINUTE // arbitrary
+export const interactionSelectorMap = new Map<number, string | undefined>() // key is timestamp
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** We could probably have a more explicit name. Also if key is a timestamp you could use TimeStamp type
```suggestion
export const interactionSelectorByTimestamp = new Map<TimeStamp, string | undefined>()
```

Reply:
Should be fixed. Though I used RelativeTime suggested below for this type, I can see there has been confusion around this and `TimeStamp` used in legacy cases.